docs/user/environment-variables.md

```diff
@@ -38,6 +38,8 @@ The installer accepts a number of environment variable that allow the interactiv
 
 ## Platform-Specific
 
+* `AWS_PROFILE`:
```

Comment:
If the `OPENSHIFT_INSTALL_` prefix was used here, as with the other variables, it'd be consistent and also allow for a different profile used with the openshift-installer than what is generally set for the environment. The paid for that is to override `AWS_PROFILE` accordingly before calling out to terraform in the installer and also pass it to the aws-sdk-go in the code somehow.